### PR TITLE
Delete useless parameter from the e2e test function  in [node_problem_detector.go]

### PR DESCRIPTION
**Pull request by heqg**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
Delete useless parameter from function

Which issue(s) this PR fixes:

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

```release-note
NONE
```
Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

### CODE REVIEW

File: test/e2e/node/node_problem_detector.go [hunk 1]
```diff
@@ -147,11 +147,11 @@ var _ = SIGDescribe("NodeProblemDetector [DisabledForLargeClusters]", func() {
...
-					rss, workingSet := getMemoryStat(f, host)
+					rss, workingSet := getMemoryStat(host)
 					rssStats[host] = append(rssStats[host], rss)
 					workingSetStats[host] = append(workingSetStats[host], workingSet)
 					if i == numIterations {
-						cpuUsage, uptime := getCPUStat(f, host)
+						cpuUsage, uptime := getCPUStat(host)
 						cpuUsageStats[host] = append(cpuUsageStats[host], cpuUsage)
 						uptimeStats[host] = append(uptimeStats[host], uptime)
 					}
```
Comment (Aut0R3V) [inline test/e2e/node/node_problem_detector.go:157]: I feel like your effort seems good, but I think their might be various references involving the framework. 
I'm adding in examples of the function which I think might be involving the use of the parameters that you're proposing to delete. 
func verifyEvents(f *framework.Framework, options metav1.ListOptions, num int, reason, nodeName string) error {
func verifyEventExists(f *framework.Framework, options metav1.ListOptions, reason, nodeName string) error {
func verifyNodeCondition(f *framework.Framework, condition v1.NodeConditionType, status v1.ConditionStatus, reason, nodeName string) error {

**Comment by Aut0R3V** [comment]
node, err := f.ClientSet.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})

File: test/e2e/node/node_problem_detector.go [hunk 2]
```diff
@@ -147,11 +147,11 @@ var _ = SIGDescribe("NodeProblemDetector [DisabledForLargeClusters]", func() {
...
 				if isStandaloneMode[host] {
-					rss, workingSet := getMemoryStat(f, host)
+					rss, workingSet := getMemoryStat(host)
 					rssStats[host] = append(rssStats[host], rss)
 					workingSetStats[host] = append(workingSetStats[host], workingSet)
 					if i == numIterations {
-						cpuUsage, uptime := getCPUStat(f, host)
+						cpuUsage, uptime := getCPUStat(host)
 						cpuUsageStats[host] = append(cpuUsageStats[host], cpuUsage)
 						uptimeStats[host] = append(uptimeStats[host], uptime)
```
Comment (Aut0R3V) [inline test/e2e/node/node_problem_detector.go:156]: I think the overall idea is good but there might be several references in the code where the parameters that are being proposed to be deleted might be required. More details in the conversation.

File: test/e2e/node/node_problem_detector.go [hunk 3]
```diff
@@ -239,7 +239,7 @@ func verifyNodeCondition(f *framework.Framework, condition v1.NodeConditionType,
 	return nil
 }
 
-func getMemoryStat(f *framework.Framework, host string) (rss, workingSet float64) {
+func getMemoryStat(host string) (rss, workingSet float64) {
```
Comment (Aut0R3V) [inline test/e2e/node/node_problem_detector.go:242]: Just before this, there's function where it is being called.
	node, err := f.ClientSet.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})

File: test/e2e/node/node_problem_detector.go (hunk 1 above)
Comment (Aut0R3V) [inline test/e2e/node/node_problem_detector.go:157]: For instance there's this particular code snippet :
https://github.com/kubernetes/kubernetes/blob/c5cc25d1cb075f84756db9bffecd3fafe12c5fab/test/e2e/node/node_problem_detector.go#L215

File: test/e2e/node/node_problem_detector.go [hunk 4]
```diff
@@ -239,7 +239,7 @@ func verifyNodeCondition(f *framework.Framework, condition v1.NodeConditionType,
 	return nil
```
Comment (Aut0R3V) [inline test/e2e/node/node_problem_detector.go:239]: Another example can be : 
https://github.com/kubernetes/kubernetes/blob/c5cc25d1cb075f84756db9bffecd3fafe12c5fab/test/e2e/node/node_problem_detector.go#L228

**Comment by Aut0R3V** [comment]
I've added some comments and I'm a bit uncertain about the changes. Maybe I'm wrong, it would be nice to hear from members also.

File: test/e2e/node/node_problem_detector.go (hunk 3 above)
Comment (heqg) [reply test/e2e/node/node_problem_detector.go:242]: > ode, err := f.ClientSet.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})

this particular code snippet  is in function [verifyNodeCondition], 
not  in the function of  [getCPUStat] or [getMemoryStat] ,

so, Will it also be affected if the parameter “(f *framework.Framework) in function [getCPUStat] or [getMemoryStat] ” is deleted directly?

**Comment by heqg** [comment]
Temporarily close the non-bug related PR

**Pull request closed without merging**